pandas-stubs/core/frame.pyi

```diff
@@ -2148,7 +2149,7 @@ class DataFrame(NDFrame, OpsMixin, _GetItemHack):
     def reindex_like(
         self,
         other: DataFrame,
-        method: _str | FillnaOptions | Literal["nearest"] | None = ...,
+        method: FillnaOptions | Literal["nearest"] | None = ...,
```

Comment:
ruff can now detect that `_str` shadows the literals we accept so it started complaining here